SocialCareCaseViewerApi/V1/UseCase/CaseStatusesUseCase.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+using SocialCareCaseViewerApi.V1.Boundary.Response;
+using SocialCareCaseViewerApi.V1.Exceptions;
+using SocialCareCaseViewerApi.V1.Factories;
+using SocialCareCaseViewerApi.V1.Gateways;
+using SocialCareCaseViewerApi.V1.Domain;
+
+namespace SocialCareCaseViewerApi.V1.UseCase.Interfaces
+{
+    public class CaseStatusesUseCase : ICaseStatusesUseCase
+    {
+        private IDatabaseGateway _databaseGateway;
+
+        public CaseStatusesUseCase(IDatabaseGateway databaseGateway)
+        {
+            _databaseGateway = databaseGateway;
+        }
+
+        public ListCaseStatusesResponse ExecuteGet(long personId)
+        {
+            var caseStatus = _databaseGateway.GetCaseStatusesByPersonId(personId);
+
+            if (caseStatus == null || caseStatus.Count == 0)
```

Comment:
can `caseStatus` be null? I can see the Count being 0 but wouldn't it always be an Object?

Reply:
Good catch, caseStatus is never null (worst case it's an empty list)